templates/CRM/Banking/Page/Review.tpl

```diff
@@ -188,6 +202,57 @@ function analysePayment() {
   );
 }
 
+/**
+ * Resets a transaction into the status "New"
+ */
+function resetPayment() {
+  const response = confirm('{/literal}{ts domain='org.project60.banking'}The analysis of this transaction will be reset into its initial state and all saved suggestions will be reset. Are you sure you want to do that?{/ts}{literal}');
+  if (!response) {
+    return;
+  }
+
+  var reload_regex = new RegExp("(execute=)[0-9]+", 'ig');
```

Comment:
The use of double and single quotes isn't consistent. (Here and in the rest of the code.)